### DISCUSSION TITLE
corewcf.primitives\1.5.1\analyzers\dotnet\roslyn4.0\cs\CoreWCF.BuildTools.Roslyn4.0.dll : Could not load file or assembly 'Microsoft.CodeAnalysis, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified..

### DISCUSSION THREAD

**Discussion by firuzan-degroof-tfs** [CoreWCF/CoreWCF#1325] [General]
Hi, we are building our code with VS2022 and started to have some issue please do you have any idea to solve this issue.  We changed from Microsof.CodeAnaysis to Microsoft.CodeAnalysis.NetAnalyzers. It is not solution.

**Comment by g7ed6e** [comment]
Hi @firuzan-degroof-tfs can you share a sample project having this issue ?

**Reply by firuzan-degroof-tfs** [reply]
I don't have a sample project. I didn't get this error local. I have in the build server this issue.
